src/fhirConfig.ts

```diff
@@ -23,6 +23,7 @@ export interface Auth {
 
 export interface Server {
     url: string;
+    hasCORS?: boolean;

```

Comment:
What does this mean? CORS is not a boolean concept. It requires configuration parameters, most notably the allowed origins (for `Access-Control-Allow-Origin` header)

Reply:
Yep your right -- my mistake this is in conjunction with this PR: https://github.com/awslabs/fhir-works-on-aws-routing/pull/14/files